src/nidigital/examples/burst_with_start_trigger/nidigital_burst_with_start_trigger_grpc.py

```diff
@@ -0,0 +1,76 @@
+
+        # Apply the settings from the levels and timing sheets we just loaded to the session
+        session.apply_levels_and_timing(levels_filename, timing_filename)
+
+        # Loading the pattern file (.digipat) created using the Digital Pattern Editor
+        pattern_filename = os.path.join(dir, 'Pattern.digipat')
+        session.load_pattern(pattern_filename)
+
+        if trigger_source is None:
+            print('Start bursting pattern')
+        else:
+            # Specify a source and edge for the external start trigger
+            session.start_trigger_type = nidigital.TriggerType.DIGITAL_EDGE
+            session.digital_edge_start_trigger_source = trigger_source
+            session.digital_edge_start_trigger_edge = nidigital.DigitalEdge.RISING if trigger_edge == 'Rising' else nidigital.DigitalEdge.FALLING
+            print('Wait for start trigger and then start bursting pattern')
+
+        # If start trigger is configured, waiting for the trigger to start bursting and then blocks until the pattern is done bursting
+        # Else just start bursting and block until the pattern is done bursting
+        session.burst_pattern(start_label='new_pattern')
+
+        # Disconnect all channels using programmable onboard switching
+        session.selected_function = nidigital.SelectedFunction.DISCONNECT
+    print('Done bursting pattern')
+
+
+def _main(argsv):
+    parser = argparse.ArgumentParser(description='Demonstrates how to create and configure a session that bursts a pattern on the digital pattern instrument using a start trigger', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2,PXI1Slot3', help='Resource name of a NI digital pattern instrument. Ensure the resource name matches the instrument name in the pinmap file.')
+    parser.add_argument('-s', '--simulate', default='True', choices=['True', 'False'], help='Whether to run on simulated hardware or real hardware')
+    parser.add_argument('-a', '--address', default='localhost', help='Server address.' )
+    parser.add_argument('-p', '--port', default='31763', help='Server port.')
+    subparser = parser.add_subparsers(dest='command', help='Sub-command help')
+    start_trigger = subparser.add_parser('start-trigger', help='Configure start trigger')
+    start_trigger.add_argument('-ts', '--trigger-source', default='/PXI1Slot2/PXI_Trig0', help='Source terminal for the start trigger')
+    start_trigger.add_argument('-te', '--trigger-edge', default='Rising', choices=['Rising', 'Falling'], help='Trigger on rising edge or falling edge of start trigger')
+    args = parser.parse_args(argsv)
+
+    example(args.resource_name,
+            'Simulate=1, DriverSetup=Model:6571' if args.simulate == 'True' else '',
+            args.trigger_source if args.command == 'start-trigger' else None,
+            args.trigger_edge if args.command == 'start-trigger' else None, args.address, args.port )
```

Comment:
When editing code, you must match existing style (or in rare cases be changing an entire file to match a new standard, but that's not the case here). You can see each argument to this method is on a separate line. Your new arguments should do that too.